Specifications/QIR/Data-Types.md

```diff
@@ -2,15 +2,13 @@
 
```

Comment:
Can you please also address the following?

1. I feel the discrepancy for the PaulyY and PaulyZ values between the fragment 
> 0 is PauliI, 1 is PauliX, _**3**_ is PauliY, and _**2**_ is PauliZ

and the fragment (sorry, the `@` causes some unexpected formatting)
> @PauliI = constant i2 0
@PauliX = constant i2 1
@PauliY = constant i2 _**-1**_
@PauliZ = constant i2 _**-2**_
2. I propose the following change.
```diff
2c2
< 0..2..4 = {0. 2. 4}
---
> 0..2..4 = {0, 2, 4}
```

Reply:
I understood the item 1. I propose the following clarification in [this file](https://github.com/microsoft/qsharp-language/blob/main/Specifications/QIR/Data-Types.md):
```diff
@@ -86,8 +86,8 @@ The following global constants are defined for use with the `%Result` and `%Paul
 
 @PauliI = constant i2 0
 @PauliX = constant i2 1
-@PauliY = constant i2 -1
-@PauliZ = constant i2 -2
+@PauliY = constant i2 -1    ; The value 3 (binary 11) is displayed as a 2-bit sigend value of -1 (binary 11).
+@PauliZ = constant i2 -2    ; The value 2 (binary 10) is displayed as a 2-bit sigend value of -2 (binary 10).
```

Reply:
Thanks for catching the dot, and I added the comment.